Week1/github.js

```diff
@@ -0,0 +1,94 @@
+/* global createAndAppend, fetchJSON */
+'use strict';
+{
+    const URL_Rep = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+    function renderRepository(container, repository) {
+        const dateOfRepository = new Date(repository.updated_at);
+        container.innerHTML = "";
+        const innerTable = createAndAppend("table", container);
+        const tBody = createAndAppend("tbody", innerTable);
+        const tr = createAndAppend('tr', tBody);
+        createAndAppend('td', tr, {text: "Repository:", class: "nameRow"});
+        const tdForLink = createAndAppend('td', tr,);
+        createAndAppend('a', tdForLink, {text: repository.name, href: repository.html_url});
+        const tr1 = createAndAppend('tr', tBody);
+        createAndAppend('td', tr1, {text: "Description:", class: "nameRow"});
+        createAndAppend('td', tr1, {text: repository.description});
+        const tr2 = createAndAppend('tr', tBody);
+        createAndAppend('td', tr2, {text: "Forks:", class: "nameRow"});
+        createAndAppend('td', tr2, {text: repository.forks});
+        const tr3 = createAndAppend('tr', tBody);
+        createAndAppend('td', tr3, {text: "Updated:", class: "nameRow"});
+        createAndAppend('td', tr3, {text: dateOfRepository.toLocaleString()});
+
+
+    }
+    //contributors box
+    async function renderContributors(containerForContributors, repository) {
+        const contributorsURL = repository['contributors_url'];
+        try {
+            const contributors = await fetchJSON(contributorsURL);
+                containerForContributors.innerHTML = "";
```

Comment:
Your indentation is off. Please make sure your document is well-formatted. Check the VSCode tips for helpful settings.

https://github.com/HackYourFuture/fundamentals/tree/master/VSCodeTips#customise-vscode-settings